apps/autonolas-registry/common-util/functions/ifps.tsx

```diff
@@ -0,0 +1,49 @@
+import { notifyError } from '@autonolas/frontend-library';
+import { GATEWAY_URL, HASH_PREFIX } from '../../util/constants';
```

Comment:
tiny, could use from libs/util-constants/src

Reply:
i just realised file name is wrong 😭